Mathlib/CategoryTheory/Monad/Monadicity.lean

```diff
@@ -282,6 +272,10 @@ variable [IsRightAdjoint G]
 
 section BeckMonadicity
 
+-- Porting note: these local notations were not used in this section before
```

Comment:
This should have not leaked. It was my fault in #5088. Fixed in #5830. Same below